tests/integration/conftest.py

```diff
@@ -64,4 +83,60 @@ def time_series(request, cognite_client):
     create_subscription_in_cdf(timeseries_set, sub_name, cognite_client)
     yield timeseries_set
     remove_time_series_data(timeseries_set, cognite_client)
-    cognite_client.time_series.subscriptions.delete(sub_name)
\ No newline at end of file
+    cognite_client.time_series.subscriptions.delete(sub_name)
+
+@pytest.fixture(scope="session")
+def test_space(test_config, cognite_client: CogniteClient):
+    space_id = test_config["data_modeling"][0]["space"]
+    space = cognite_client.data_modeling.spaces.retrieve(space_id)
+    if space is None:
+        new_space = SpaceApply(space_id, name="Integration Test Space", description="The space used for integration tests.")
+        space = cognite_client.data_modeling.spaces.apply(new_space)
+    else: # Ensure there aren't existing data models
+        data_model_list = cognite_client.data_modeling.data_models.list(space=space_id)
+        assert len(data_model_list) == 0, "Space should not have existing data models, please remove models/views/containers before testing"
+    yield space
+    cognite_client.data_modeling.spaces.delete(spaces=[space_id])
+    
+
+@pytest.fixture(scope="session")
+def test_dml():
+    return (RESOURCES / "movie_model.graphql").read_text()
+
+@pytest.fixture(scope="function")
+def test_model(cognite_client: CogniteClient, test_space: Space, test_dml: str):
+    movie_id = DataModelId(space=test_space.space, external_id="Movie", version="1")
+    created = cognite_client.data_modeling.graphql.apply_dml(
+        id=movie_id, dml=test_dml, name="Movie Model", description="The Movie Model used in Integration Tests"
+    )
+    models = cognite_client.data_modeling.data_models.retrieve(created.as_id(), inline_views=True)
+    yield models.latest_version()
+    cognite_client.data_modeling.data_models.delete(ids=[movie_id])
+    views = models.data[0].views
+    for view in views: # Views and containers need to be deleted so the space can be deleted
+        cognite_client.data_modeling.views.delete((test_space.space, view.external_id, view.version))
+        cognite_client.data_modeling.containers.delete((test_space.space, view.external_id))
+
+@pytest.fixture(scope="function")
+def edge_table_name(test_space, azure_credential):
+    edge_table_name = lakehouse_table_name(test_space.space + "_edges")
+    yield edge_table_name
+    try:
+        delta_table = get_ts_delta_table(azure_credential, edge_table_name)
+        delta_table.delete()
+    except TableNotFoundError:
+        print(f"Table not found {edge_table_name}")
+
+@pytest.fixture(scope="function")
+def view_tables(test_space, azure_credential):
```

Comment:
Why do we need the table for views given that the implementation of dm replicator did not write views to a Lakehouse table?

Reply:
Could you clarify this for me?  When running the replicator in its current state using the cc_plant data model, view tables were written to my Fabric lakehouse.

Reply:
I thought we have tables for nodes and tables for edges, but we don't have tables of views based off which the nodes and edges are implementing.

Reply:
Discussed offline, while the views are where the table names come from the tables are representative of the data model instances.  Will rename to reflect.